IdentityCore/src/MSIDOAuth2Constants.m

```diff
@@ -142,6 +144,8 @@
 NSString *const MSID_APP_METADATA_AUTHORITY_MAP_TYPE     = @"authority_map";
 NSString *const MSID_DEVICE_ID_CACHE_KEY                 = @"device_id";
 NSString *const MSID_PRT_PROTOCOL_VERSION_CACHE_KEY      = @"prt_protocol_version";
+NSString *const MSID_KID_CACHE_KEY                       = @"kid";
```

Comment:
This field is called "key_id" in the shared schema. Please update for compatibility.

Reply:
The cache schema for sample access token value has "kid" so leaving this as is.

Reply:
It says key_id here: https://identitydivision.visualstudio.com/DevEx/_git/AuthLibrariesApiReview?path=%2FSSO%2FSchema.md&_a=preview&anchor=access-token-schema
Can you confirm which one is right?